src/components/ShareTrajectoryModal/EmbedSnippetPanel.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import { Input, Checkbox, Button } from "antd";
+import { URL_PARAM_KEY_FILE_NAME } from "../../constants";
+import VerticalFlexbox from "../../styles/utils";
+import { getUrlParamValue, copyToClipboard } from "../../util";
+import { DownArrow } from "../Icons";
+
+import styles from "./style.css";
+
+const DEFAULT_WIDTH = 834;
+const DEFAULT_HEIGHT = 480;
+const MAX_WIDTH = 3840;
+const MAX_HEIGHT = 2160;
```

Comment:
if something is an unchanged constant, just defining them like this is clearer than having an object (which makes it less clear something is not supposed to be edited because attributes can be changes, while `const` can't be)